lxd/db/cluster/identities.go

```diff
@@ -497,6 +497,30 @@ func ActivateClusterLinkIdentity(ctx context.Context, tx *sql.Tx, identifier uui
 	return nil
 }
 
+var getPendingClusterLinkIdentityByTokenSecretStmt = fmt.Sprintf(`
```

Comment:
Discussed in 1:1 - Can move `getPendingTLSIdentityByTokenSecretStmt` into `GetPendingTLSIdentityByTokenSecret`, then factor the identity type out so the same function can be used for both.